### ISSUE TITLE
WOFF2-only CSS option

### ISSUE THREAD

**Issue by Timmmm** [majodev/google-webfonts-helper#162]
"Modern browsers" includes WOFF even though global support for WOFF2 is 97%. Given that they're only fonts and it's hardly a disaster if they don't load, I think that option should ditch WOFF and have WOFF2 only.

**Comment by Timmmm** [comment]
"Modern browsers" also shouldn't include `local('')` which is apparently for IE6-8 and can break Android: #84

**Comment by majodev** [comment]
v1.1.0 removes`local('')`. However, for sanity I'll keep `.woff` in for now.

**Comment by Timmmm** [comment]
Nice. What do you mean "for sanity"?

**Comment by thestinger** [comment]
Internet Explorer 11 isn't supported anymore so I really don't think including woff is needed for a modern browser option.

https://caniuse.com/?search=woff

**Comment by gitressa** [comment]
I would also prefer it, if there were a `.woff2`-only option. I usually remove `.woff`-files to reduce complexity and size, and just use `.woff2`.

Example, with [12 Roboto fonts](https://gwfh.mranftl.com/fonts/roboto):

| Versions | Files | Size |
| --- | --- | --- |
| `.woff` +  `.woff2` | 24 | 453,5 kB |
| `.woff2` | 12 | 198,6 kB |

**Comment by Rotzbua** [comment]
The support modes should be renamed to reflect the current state of technology:

| Old label | New label          | Format |
| -- | -- | -- |
| - | Modern Browser | woff2 |
| Modern Browser | Best Support   | woff2 + woff |
| Best Support | Legacy Support | woff2 + woff + ttf + svg + eot |

**Comment by thestinger** [comment]
IE11 is no longer supported and is disabled automatically on up-to-date Windows. IE11 has woff1 support and was the oldest browser nearly anyone was interested in supporting. I don't see much of a reason to have anything but woff2. It's not as if the website won't work in IE11 without web fonts anyway. Browsers already won't load them in such cases on a slow / metered connection.

**Comment by thestinger** [comment]
IE11 is no longer supported and is disabled automatically on up-to-date Windows. IE11 has woff1 support and was the oldest browser nearly anyone was interested in supporting. I don't see much of a reason to have anything but woff2. Sites are meant to work without the web fonts being loaded anyway. There are many reasons for modern sites not working in IE11 and not having the right fonts isn't a major issue.

**Comment by Rotzbua** [comment]
I have reconsidered my previous suggestion and think the term "best" does not reflect the development status and is very subjective. I think the terms "legacy" and "historic" support are more accurate.

For legacy support, as before suggest, the most obvious format would be `woff`.  Taking the CanIUse's percentages into account, `ttf` seems a better candidate. `ttf` supports more browsers and the uncompressed source form of `woff(2)`. If necessary the compression can be applied by http transport compression gzip if a legacy browsers has support.

Adding the percentages of OperaMini (does not support any custom font) current global usage is:

| Format | %     | caniuse                           |
|--------|-------|-----------------------------------|
| woff2  | 98,15 | [link](https://caniuse.com/woff2) |
| woff   | 99,62 | [link](https://caniuse.com/woff)  |
| ttf    | 99,80 | [link](https://caniuse.com/ttf)   |

In summary my updated suggestion would be:

| Old label | New label          | Format |
| -- | -- | -- |
| - | Modern Browser | woff2 |
| Modern Browser | Legacy Support   | woff2 + ttf |
| Best Support | Historic Support | woff2 + ttf + woff + svg + eot |

And the website would look like:

![fonts](https://github.com/majodev/google-webfonts-helper/assets/7337347/cbb79bef-845a-4a92-b965-5169b92ef830)

**Comment by gitressa** [comment]
Perfect, thanks @Rotzbua and @majodev!
